pkg/frontend/admin_platformworkloadidentityroleset_list_test.go

```diff
@@ -14,7 +14,7 @@ import (
 	testdatabase "github.com/Azure/ARO-RP/test/database"
 )
 
-func TestPlatformWorkloadIdenittyRoleSetList(t *testing.T) {
```

Comment:
I keep doing the same thing :)

Reply:
I caught this one and fixed it myself before opening the PR. I think you're commenting on the content of one of my older commits.